src/main/kotlin/apply/application/ApplicationFormService.kt

```diff
@@ -60,4 +61,70 @@ class ApplicationFormService(
         )
         applicationFormRepository.saveAll(applicationForms)
     }
+
+    fun save(applicantId: Long, applicationFormSaveRequest: ApplicationFormSaveRequest) {
+        if (applicationFormRepository.existsByRecruitmentIdAndApplicantId(applicationFormSaveRequest.recruitmentId, applicantId)) {
+            throw IllegalArgumentException("이미 저장된 지원서가 있습니다.")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(applicationFormRepository.existsByRecruitmentIdAndApplicantId(applicationFormSaveRequest.recruitmentId, applicantId)) {
            "이미 저장된 지원서가 있습니다."
        }
```

Reply:
```        require(!applicationFormRepository.existsByRecruitmentIdAndApplicantId(applicationFormSaveRequest.recruitmentId, applicantId)) {
            "이미 저장된 지원서가 있습니다."
        }
```

와우... 코틀린이 제공하는 거죠? 신세계네요 :)

Reply:
[check과 assesrt](https://www.bignerdranch.com/blog/write-better-code-using-kotlins-require-check-and-assert/)라는 것도 있네요!

Reply:
@toneyparky 
`assert`는 Java의 `assert`와 동일합니다. 디버깅 목적으로 사용해요.